src/gsy_e/models/strategy/scm/load.py

```diff
@@ -87,9 +90,11 @@ def get_available_energy_kWh(self, time_slot: DateTime) -> float:
 
     def decrease_energy_to_buy(
             self, traded_energy_kWh: float, time_slot: DateTime, area: "AreaBase") -> None:
-        """Decrease traded energy from the state and the strategy parameters."""
-        self._energy_params.state.decrement_energy_requirement(
-            traded_energy_kWh, time_slot, area.name)
+        """Decrease the amount of traded energy from the asset's state."""
+        self._energy_params.decrement_energy_requirement(
+            energy_kWh=traded_energy_kWh,
+            time_slot=time_slot,
+            area_name=area.name)
```

Comment:
Same as https://github.com/gridsingularity/gsy-e/pull/1523/files#r963795487.

The `state.decrement_energy_requirement()` method expects Wh values (not kWh).